examples/scripts/blender/sdf_exporter.py

```diff
@@ -1,235 +1,217 @@
-import xml.etree.ElementTree as ET
-from os import path
-from xml.dom import minidom
-
 import bpy
-from bpy.props import StringProperty
-from bpy.types import Operator
+import os.path
 from bpy_extras.io_utils import ImportHelper
+from bpy.props import StringProperty, BoolProperty
+from bpy.types import Operator
 
-# Tested Blender version: 2.82/3.2
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
 
-########################################################################################################################
-### Exports model.dae of the scene with textures, its corresponding model.sdf file, and a default model.config file ####
-########################################################################################################################
 def export_sdf(prefix_path):
-
-    dae_filename = "model.dae"
-    sdf_filename = "model.sdf"
-    model_config_filename = "model.config"
-    lightmap_filename = "LightmapBaked.png"
-    model_name = "my_model"
-    meshes_folder_prefix = "meshes/"
-
+    dae_filename = 'model.dae'
+    sdf_filename = 'model.sdf'
+    model_config_filename = 'model.config'
+    lightmap_filename = 'LightmapBaked.png'
+    model_name = 'my_model'
+    meshes_folder_prefix = 'meshes/'
+    
     # Exports the dae file and its associated textures
-    bpy.ops.wm.collada_export(
-        filepath=path.join(prefix_path, meshes_folder_prefix, dae_filename),
-        check_existing=False,
-        filter_blender=False,
-        filter_image=False,
-        filter_movie=False,
-        filter_python=False,
-        filter_font=False,
-        filter_sound=False,
-        filter_text=False,
-        filter_btx=False,
-        filter_collada=True,
-        filter_folder=True,
-        filemode=8,
-    )
+    bpy.ops.wm.collada_export(filepath=prefix_path+meshes_folder_prefix+dae_filename, 
+                             check_existing=False, 
+                             filter_blender=False, 
+                             filter_image=False, 
+                             filter_movie=False, 
+                             filter_python=False, 
+                             filter_font=False, 
+                             filter_sound=False, 
+                             filter_text=False, 
+                             filter_btx=False, 
+                             filter_collada=True, 
+                             filter_folder=True, 
+                             filemode=8)
 
-    # objects = bpy.context.selected_objects
     objects = bpy.context.selectable_objects
-    mesh_objects = [o for o in objects if o.type == "MESH"]
-    light_objects = [o for o in objects if o.type == "LIGHT"]
-
-    #############################################
-    #### export sdf xml based off the scene #####
-    #############################################
-    sdf = ET.Element("sdf", attrib={"version": "1.8"})
+    mesh_objects = [o for o in objects if o.type == 'MESH']
```

Comment:
Please revert unnecessary changes. This is only changing double quotes to single quotes. The rest of the file uses double quotes and I think it's best to stick with that.